tests/dicom/sort/test_sort_utils.py

```
@@ -0,0 +1,161 @@
+		with pytest.raises(ValueError):
+			result = read_tags(
+				file=dicom_test_file,
+				tags=tags,
+				truncate=True,
+				sanitize=True
+			)
+
+	def test_read_tags_with_partial_metadata(self, dicom_test_file):
+		tags = ['PatientName', 'Modality', 'NonexistentTag']
+		with pytest.raises(ValueError):
+			result = read_tags(
```

Ruff

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)